src/nano_contracts/serializer.ts

```diff
@@ -160,7 +161,7 @@ class Serializer {
    * @memberof Serializer
    * @inner
    */
-  fromOptional(value: any, type: string): Buffer {
+  fromOptional(value: NanoContractArgumentType, type: string): Buffer {
```

Comment:
This `value` can be `null` as well

Reply:
This was solved by editing the type directly, as said above.